apps/browser/lib/hooks/useEvoService.ts

```diff
@@ -73,21 +71,26 @@ export const useEvoService = (
         title: "Reviewing your prompt..."
       }]);
     } else {
-      setChatLogState(chatLog);
+      setChatLogState([...chatLog]);
```

Comment:
isn't this the same as `setChatLogState(chatLog)` ?

Reply:
This forces a reset of the state variable in case state is the same instance, but could probably be removed. I was just paranoid while debugging so did this to avoid it being a source of not having a rerender.